src/api/app/helpers/event_mailer_helper.rb

```diff
@@ -1,12 +1,6 @@
 module EventMailerHelper
   def project_or_package_text(project, package, opts = {})
-    if package.present?
-      text = "#{project}/#{package}"
-    else
-      text = project
-    end
-    return unless opts[:short].nil?
-    return "package #{text}" if package.present?
-    "project #{text}"
+    return "#{opts[:short].nil? ? 'package ' : ''}#{project}/#{package}" if package.present?
+    "#{opts[:short].nil? ? 'project ' : ''}#{project}"
```

Comment:
Yes. Didn't want to change the functionality even if it's unused 👍But I strongly think we should get rid of it 😄